easybuild/easyconfigs/s/snakemake/snakemake-6.1.0-foss-2020b.eb

```diff
@@ -0,0 +1,85 @@
+easyblock = 'PythonBundle'
+
+name = 'snakemake'
+version = '6.1.0'
+
+homepage = 'https://snakemake.readthedocs.io'
+description = "The Snakemake workflow management system is a tool to create reproducible and scalable data analyses."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('PyYAML', '5.3.1'),
+    ('GitPython', '3.1.14'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+# snakemake-6.1.0_fix_jobs.patch is needed for clusters that do not support copying the full env into the batch job
+exts_list = [
+    ('wrapt', '1.12.1', {
+        'checksums': ['b62ffa81fb85f4332a4f609cab4ac40709470da05643a082ec1eb88e6d9b97d7'],
+    }),
+    ('ratelimiter', '1.2.0.post0', {
+        'checksums': ['5c395dcabdbbde2e5178ef3f89b568a3066454a6ddc223b76473dac22f89b4f7'],
+    }),
+    ('ConfigArgParse', '1.4', {
+        'checksums': ['abef9ff44fb0091f0e3bb2ee7e5b26a02b5b62d45408a5272a9bd461f5b59b4b'],
+    }),
+    ('datrie', '0.8.2', {
+        'checksums': ['525b08f638d5cf6115df6ccd818e5a01298cd230b2dac91c8ff2e6499d18765d'],
+    }),
+    ('pyrsistent', '0.17.3', {
+        'checksums': ['2e636185d9eb976a18a8a8e96efce62f2905fea90041958d8cc2a189756ebf3e'],
+    }),
+    ('traitlets', '5.0.5', {
+        'checksums': ['178f4ce988f69189f7e523337a3e11d91c786ded9360174a3d9ca83e79bc5396'],
+    }),
+    ('ipython_genutils', '0.2.0', {
+        'checksums': ['eb2e116e75ecef9d4d228fdc66af54269afa26ab4463042e33785b887c628ba8'],
+    }),
+    ('jupyter-core', '4.7.1', {
+        'modulename': 'jupyter_core',
+        'source_tmpl': 'jupyter_core-%(version)s.tar.gz',
+        'checksums': ['79025cb3225efcd36847d0840f3fc672c0abd7afd0de83ba8a1d3837619122b4'],
+    }),
+    ('nbformat', '5.0.8', {
```

Comment:
Maybe we could you maybe use IPython as a dependency instead? It does add a bunch of additional stuff we might not need, but it would pick up traitlets, ipython_genutils, jupyter-core, nbformat

Reply:
ok, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/12557/commits/8e5882ee1c13dac09796883c4cdb6d362b2e72d7